src/transaction/log_tran_table.c

```diff
@@ -1473,6 +1475,48 @@ logtb_free_tran_mvcc_info (LOG_TDES * tdes)
   curr_mvcc_info->sub_ids.clear ();
 }
 
+#if defined (SERVER_MODE)
+/*
+ * logtb_set_session_tdes - reflect the session to the transaction
+ *
+ * return: nothing
+ *
+ *   tdes(in/out): Transaction descriptor
+ */
+static void
+logtb_set_session_tdes (THREAD_ENTRY * thread_p, LOG_TDES * tdes)
+{
+  SESSION_PARAM *param;
+  int wait_msecs;
+
+  /* isolation level */
+  param = session_get_session_parameter (thread_p, PRM_ID_LOG_ISOLATION_LEVEL);
+  if (param)
+    {
+      tdes->isolation = (TRAN_ISOLATION) param->value.i;
+    }
+
+  /* lock timeout */
+  param = session_get_session_parameter (thread_p, PRM_ID_LK_TIMEOUT);
+  if (param)
+    {
+      wait_msecs = param->value.i;
+      if (wait_msecs > 0)
+	{
+	  tdes->wait_msecs = wait_msecs * 1000;
+	}
+      else if (wait_msecs == LK_ZERO_WAIT || wait_msecs == LK_INFINITE_WAIT || wait_msecs == LK_FORCE_ZERO_WAIT)
+	{
+	  tdes->wait_msecs = wait_msecs;
+	}
+      else
+	{
+	  assert_release (0);
+	}
+    }
```

Comment:
```suggestion
  if (param)
    {
      tdes->wait_msecs = param->value.i;
      if (tdes->wait_msecs > 0)
	{
	 tdes->wait_msecs = tdes->wait_msecs * 1000;
	}
    }
```

Reply:
or 
```suggestion
  if (param)
    {
       wait_msecs = param->value.i;
       tdes->wait_msecs = wait_msecs > 0 ? wait_msecs * 1000 : wait_msecs; 
    }
```

Reply:
The `PRM_ID_LK_TIMEOUT` can't be under -1, so `assert_release (0);` doesn't seem needed. The suggestions above are just suggestions.

Reply:
I changed param condition (NULL check) to assert because I thought the session paramter `PRM_ID_LOG_ISOLATION_LEVEL`, `PRM_ID_LK_TIMEOUT` should be unconditionally valid.

It's right. `assert_release (0)` seems unnecessary. I will remove `assert_release (0)`.